docs/security.md

```diff
@@ -0,0 +1,68 @@
+# Securing KUDO Cassandra Operator instances
+
+The KUDO Cassandra service supports Cassandra’s native transport **encryption**

```

Comment:
Let's rephrase all instances of "service" with "operator" to maintain consistency.
```suggestion
The KUDO Cassandra operator supports Cassandra’s native transport **encryption**
```